autopush/memusage.py

```diff
@@ -24,11 +24,11 @@ def trap_err(func, *args, **kwargs):
     buf = StringIO()
     rusage = trap_err(resource.getrusage, resource.RUSAGE_SELF)
     buf.writelines([repr(rusage), '\n\n'])
-    trap_err(objgraph.show_most_common_types, limit=0, file=buf)
-    buf.write('\n\n')
     trap_err(pmap_extended, buf)
     trap_err(dump_rpy_heap, buf)
     trap_err(get_stats_asmmemmgr, buf)
+    buf.write('\n\n')
+    trap_err(objgraph.show_most_common_types, limit=0, file=buf)
```

Comment:
nit: since it's important that this be the last call, might want to add a comment to say that.